app/views/handlers/question.py

```diff
@@ -134,17 +135,14 @@ def get_return_to_hub_url(self):
         ):
             return url_for(".get_questionnaire")
 
-    def _get_page_title(self, transformed_block):
-        question = transformed_block.get("question")
-        if question:
-            if isinstance(question["title"], str):
-                question_title = question["title"]
-            else:
-                question_title = question["title"]["text"]
-
-            page_title = f'{question_title} - {self._schema.json["title"]}'
+    def _get_page_title(self, question):
+        question_title = (
+            question["title"]
+            if isinstance(question["title"], str)
+            else question["title"]["text"]
+        )
```

Comment:
When I run the test schema I get an error here on the `confirm-number-of-people` page:

```
...
File "/Users/andrew/dev/eq-questionnaire-runner/app/views/handlers/question.py", line 142, in _get_page_title
    else question["title"]["text"]
KeyError: 'text'
```

Reply:
This is down to not knowing what to render for a plural title until it has passed through the renderer. It is conflated with placeholder rendering. As discussed, I've added a workaround to use 'other' in titles (which should always be there).